update/command.py

```diff
@@ -12,7 +12,4 @@ def command(shell: DTShell, args):
         if shell.local_commands_info.leave_alone:
             dtslogger.warn("Will not update the commands because the path was set explicitly.")
         else:
-            if shell.update_commands():
-                dtslogger.info("Duckietown Shell commands updated.")
-            else:
-                dtslogger.error("Update was not successful.")
+            shell.check_commands_update()
```

Comment:
This is going to break if we merge it as is.
As soon as we merge this, my shell will auto-update the commands while my shell will not update until I manually do `pip3 install -U ...`. This will fail as my old shell will not have the function `check_commands_update()`.

Reply:
Reverted and retested, thank you! Lmk if you have thoughts about not having any info messages at the command level as well. It seems bare, but repeating information from the actual update functionality was clogging up the output.